tests/integration/testlib/__init__.py

```diff
@@ -15,6 +15,38 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 
+
+import os
+
 import pytest
 
+PATH_MAX = 4096
+
+
 pytest.register_assert_rewrite(__name__ + '.assertlib')
+
+
+def find_examples_dir():
```

Comment:
I would prefer you move it to examplelib.py

Reply:
Why? It seems to be a little bit too much to create a separate file for just one method.

Reply:
because it is a bad practice to define non package common helpers in __init__ files. It may cause a dependency loop in the future.
I prefer to have there nothing, package constants or something that is needed to serve the whole package. Yiu may end up having many unrelated small helpers in there, which is a problem by itself.

Reply:
Those are all theoretical problems that we do not have right now...

Reply:
If there is no super good reason, I prefer not to add code to the init files.

Reply:
moved it